finalProject/fizzBuzz.js

```diff
@@ -0,0 +1,32 @@
+
+/*Create a file named `fizzBuzz.js` and within that file, write a program that logs the numbers from 1 to 100
+ with the following rules:  
+* For any number that is a multiple of 3, log 'Fizz'
+* For any number that is a multiple of 5, log 'Buzz'
+* For any number that is a multiple of both 3 and 5, log 'FizzBuzz'
+* For all other numbers, log the number.
+
+The output of you program will look something like this:
+```
+=> 1, 2, Fizz, 4, Buzz, Fizz, 7, 8, Fizz, Buzz, 11, Fizz, 13, 14, FizzBuzz, ..., 98, Fizz, Buzz
+```
+*/
+
+var fizz = [1]
```

Comment:
It looks like this variable is not used.

Reply:
I think it was left over script from when i was first figuring out how to solve the problem and as i kept testing the file it didn't show up or harm the solution so it was never deleted.